src/opm/io/eclipse/EclUtil.cpp

```diff
@@ -27,6 +27,8 @@
 #include <fstream>
 #include <cstring>
 
+//temporary
+#include <iostream>
```

Comment:
Comment says temporary?

Reply:
thanks, used for debugging. Then I added a comment so that I should remember to remote this when I was done. Not very successful  on that.